src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private Color[] colors = Color.values();
+
     public String getRandomColor() {
-        return null;
+        return colors[new Random().nextInt(colors.length)].toString();
```

Comment:
```suggestion
        return colors[new Random().nextInt(colors.length)].name();
```
checklist
<img width="520" alt="Screen Shot 2022-10-31 at 9 01 06 PM" src="https://user-images.githubusercontent.com/90692308/199088630-8075b539-7ff6-4ff8-b76e-12b35461a92f.png">